src/Object/Post.php

```diff
@@ -346,10 +346,10 @@ public function getTemplateData(array $conv_responses, $thread_level = 1)
 			$buttons['like']    = [DI::l10n()->t("I like this \x28toggle\x29")      , DI::l10n()->t("like")];
 			$buttons['dislike'] = [DI::l10n()->t("I don't like this \x28toggle\x29"), DI::l10n()->t("dislike")];
 			if ($shareable) {
-				$buttons['share'] = [DI::l10n()->t('Share this'), DI::l10n()->t('Share')];
+				$buttons['share'] = [DI::l10n()->t('Quote and share this'), DI::l10n()->t('Quoted Share')];
```

Comment:
A "quoted share" has a specific meaning in English finance,
```suggestion
				$buttons['share'] = [DI::l10n()->t('Quote and share this'), DI::l10n()->t('Quote Share')];
```